AGGREGATE.md

```diff
@@ -551,6 +562,48 @@ as when a trigger context ID is set.
 
 See [flexible_filtering.md](https://github.com/patcg-individual-drafts/private-aggregation-api/blob/main/flexible_filtering.md) for more details.
 
+### Optional: aggregatable buckets
+
+Source registration will accept an optional field
+`aggregatable_bucket_max_budget`, which is the dictionary used to set the
+maximum contribution for each aggregatable bucket for this source.
+
+```jsonc
+{
+  ...
+  "aggregatable_bucket_max_budget": {
+    "bucket1": 32768,  // Max contribution budget for bucket1.
+    "bucket2": 32768   // Max contribution budget for bucket2.
+  }
+}
+```
+
+Trigger registration will accept an optional field `aggregatable_buckets` which
+will be used to select the contribution bucket for the generated aggregate report.
+
+```jsonc
+{
+  ...
+  "aggregatable_buckets": [
+    {
+      "bucket": "example string",
+      "filters": {"source_type": ["navigation"]}
+    }
+  ]
+}
+```
+
+The first aggregatable bucket from the trigger that matches the source filters
+will be selected. If there is no bucket specified or no matching filters, the
+`L1` contribution budget and [Aggregate Debug Reporting](aggregate_debug_reporting.md#contribution-bounding-and-budgeting)
+privacy budget will still be applied.
+
+When generating a pending aggregate report, in addition to performing the 
+current `L1` budget limit check, the required contributions for the report will
+be checked against the available budget in the selected bucket, if applicable.
+If the budget is insufficient, a debug report will be created, indicating that 
```

Comment:
Probably don’t need to call out the debug report in the explainer, as that’s covered in other documents.